how-not-to-rm-yourself.md

```diff
@@ -22,7 +22,7 @@ $ trash unicorn.png rainbow.png
 
 Even though you don't use `rm` directly, external scripts most likely will. There are some things you can do to safeguard this:
 
-- Alias `rm` to its interactive mode, which will force you to step through what it's trying to delete: `alias rm='rm -i'`
+- Alias `rm` to its interactive mode `rm -i` and set `shopt -s expand_aliases` and source your `.bashrc` in `$BASH_ENV`. This will force you to step through what it's trying to delete.
```

Comment:
Sourcing the `.bashrc` just executes it in the current process.  `.bashrc` isn't read by scripts, so if we don't tell the bash process of the script to read `.bashrc` it wont know about our alias.